Makefile

```
@@ -383,6 +492,10 @@ load-tests-swagger: start-prod wait-for-prod-health ## Execute comprehensive loa
                        ## run_smoke=true, run_average=true, run_stress=true, run_spike=true. If none set, runs all scenarios.
 	$(LOAD_TESTS_RUN_SWAGGER)
 
+test-load: load-tests ## Alias for load-tests (CRM-style naming): run K6 homepage load tests
```

checkmake (0.3.2)

[warning] Target "test-load" should be declared PHONY.

(phonydeclared)